src/js/components/Data/stories/Inline.js

```diff
@@ -33,6 +34,7 @@ const Filters = ({ search, ...rest }) => (
   <DataFilters {...rest}>
     {search && <DataSearch property="sub.name" />}
     <DataFilter property="size" />
+    <DataTableGroupBy options={['size']} />
```

Comment:
Can we add more data for this story? There's not really enough to make the grouping clear.

Also, can we add columns to the DataTable? It's a little confusing with no header cells.

Reply:
^ Agree, I think it is difficult to see group feature with such a small data set

Reply:
Good thoughts. Added a table header and more rows.